deploy/template.yaml

```diff
@@ -447,9 +447,9 @@ Resources:
             Timeout: 2
             Retries: 10 # This ensures that just one healthcheck failure won't cause the container to be drained and replaced.
             StartPeriod: 0
-      Cpu: "1024"
+      Cpu: "2048"
```

Comment:
Do you have any testfiles where we can assert that this remains true?

Reply:
Unfortunately not.